metrics.nim

```diff
@@ -968,11 +1014,7 @@ when defined(metrics) and defined(linux):
     ticks = sysconf(SC_CLK_TCK).float64
     pagesize = sysconf(SC_PAGE_SIZE).float64
 
-  method collect*(collector: ProcessInfo): Metrics =
-    let timestamp = getTime().toMilliseconds()
-    result = initOrderedTable[Labels, seq[Metric]]()
-    result[@[]] = @[]
-
+  proc updateProcessInfo() =
```

Comment:
this is a significant regression compared to the current version: instead of giving the value at collection time, it serves the value from a randomly determined past, when an unrelated metric was updated - for an application that only seldmonly (or not at all) updates other metrics, we'll see bogus values here.